sdkms/apps_generated.go

```diff
@@ -185,9 +185,37 @@ type AppCredential struct {
 	Secret               *string
 	Certificate          *Blob
 	TrustedCa            *TrustAnchor
-	GoogleServiceAccount *struct{}
+	GoogleServiceAccount *GoogleServiceAccount
```

Comment:
The type name should be `AppCredentialGoogleServiceAccount`

Reply:
updated ptal